rpcs3/Emu/FS/vfsDirBase.h

```diff
@@ -62,14 +62,12 @@ class vfsDirBase
 
 	public:
 		iterator(vfsDirBase* parent)
-			: parent(parent)
-			, data(parent->First())
+			: data(parent->First())
 		{
 		}
 
 		iterator(const DirEntryInfo* data)
-			: parent(parent)
```

Comment:
@paulsapps That assignment has no effect: http://puu.sh/f0bTL/2539451478.png (And changing it to this->parent(parent) wont change anything?)